src/math/Interpolant.js

```diff
@@ -47,7 +47,7 @@ Interpolant.prototype = {
 
 		validate_interval: {
 
-			seek: {
+			seek: while(true) {
```

Comment:
Yeah I'm rather confused by it too. IF I knew what it was supposed to do I could re-write it.